crates/alerter/src/lib.rs

```diff
@@ -250,7 +244,7 @@ impl Alerter {
             tracing::debug!(order =% uid, "found closed order");
             let start = Instant::now();
             let api_order = self.orderbook_api.order(uid).await.context("get order")?;
-            if api_order.status == OrderStatus::Fulfilled {
+            if api_order.status == Some(OrderStatus::Fulfilled) {
```

Comment:
nit: I think here it's probably a good idea to use `.unwrap()` or `.expect()` because we always expect this to be populated and would like to be informed in case this changes at some point in the future, right?

Reply:
Sure, I also thought about it.